lib/api.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const { addFiles, generateDocument } = require('./internal');
+const JSZip = require('jszip');
+
+module.exports = {
+    asBuffer,
+};
+
+async function asBuffer(html, options) {
+    const zip = new JSZip();
```

Comment:
According to this guide https://stuk.github.io/jszip/documentation/upgrade_guide.html `jszip` now works another way (without `new JSZip();`). Have you checked, is it working properly?

Reply:
Yes, I've checked. It works with `new JSZip();` https://stuk.github.io/jszip/documentation/examples.html. In that case you sent they use static method for uploading content, but in our case we use instance.